tasks/condition/condition.js

```diff
@@ -6,10 +6,17 @@
  * Task 1 - if else - z1 : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else
  * Task 2 - ternary operator - z2: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator
  */
-let z1;
-let z2;
-const x = 3;
-const y = 7;
+ const x = 3;
+ const y = 7;
+ let z1 = x + y;
```

Comment:
you should do the assignment in the scope of the following conditions and ternary operators. so, you should remove the lines 11 and 12.